examples/saml-auto-reg/README.md

```diff
@@ -0,0 +1,154 @@
+
+# WildFly image API
+Environment variables from the [WildFly image API](https://github.com/wildfly/wildfly-cekit-modules/blob/main/jboss/container/wildfly/run/api/module.yaml) that must be set in the OpenShift deployment environment
+
+* None
+
+# WildFly cloud feature-pack API
+Environment variables defined by the cloud feature-pack used to configure the server:
+
+* SSO_REALM: Keycloak realm name.
+* SSO_USERNAME=User name used to create and register the SAML client in the keycloak server.
+* SSO_PASSWORD=User password used to create and register the SAML client in the keycloak server.
+* SSO_SAML_CERTIFICATE_NAME=Alias of the private key in the SAML client keystore.
+* SSO_SAML_KEYSTORE=Client keystore file name.
+* SSO_SAML_KEYSTORE_PASSWORD=Client keystore password.
+* SSO_SAML_KEYSTORE_DIR=Directory in which the keystore is mounted.
+* SSO_SAML_LOGOUT_PAGE=URL context where to redirect when logout occurs. 
+* SSO_DISABLE_SSL_CERTIFICATE_VALIDATION=Disable keycloak server certificate check. Needed when the keycloak server generates a self signed certificate.
+* SSO_HOSTNAME_HTTPS=Route to the deployed application.
+* SSO_URL=URL of the keycloak server.
+
+# Pre-requisites
+
+* You are logged into an OpenShift cluster and have `oc` command in your path
+
+* You have installed Helm. Please refer to [Installing Helm page](https://helm.sh/docs/intro/install/) to install Helm in your environment
+
+* You have installed the repository for the Helm charts for WildFly
+
+ ```
+helm repo add wildfly https://docs.wildfly.org/wildfly-charts/
+```
+----
+**NOTE**
+
+If you have already installed the Helm Charts for WildFly, make sure to update your repository to the latest version.
+
+```
+helm repo update
+```
+----
+
+# Example steps
+
+1. Deploy an SSO server. Use the Sandbox Developer Catalog, search for sso and instantiate RH SSO 7.6 template. You can keep the default values 
+when instantiating the template. Set the admin user name `admin` and password `admin`.
+
+1. Create the SSO realm, users, role and export keystore.
+
+  * Log into the SSO admin console (`https://<SSO route>/auth/`). Use `admin` and `admin` to log-in. 
+  * Create a Realm named `WildFly`
+  * Create a Role named `user`
+  * Create a User named `demo`, password `demo`, make the password not temporary.
+  * Assign the role `user` to the user `demo`. This user will be used to log in the application.
+  * Create a User named `client-admin`, password `client-admin`, make the password not temporary. This user will be used to create 
+    the SAML client in the keycloak server. It requires more proviledges to interact with the keycloak server and to be able to create the client.
+  * In the `Client Roles` Select the Client `realm-management`, assign the role `create-client`, `manage-clients` and `manage-realm`. For latest keycloak console, select the `user` role, Click Action/Add associated roles. Then `Filter by clients`. 
```

Comment:
@jfdenise here it is not clear that the roles  need to be assigned to the `client-admin` user; perhaps the `client-admin` is not needed, in prod docs we say "Assign the roles `create-client`, `manage-clients`, and `manage-realm` to the `client-admin` user. " ... we might add a bullet point before "For latest keycloak console, "